torch/_dynamo/variables/torch.py

```diff
@@ -289,6 +291,32 @@ def call_function(
                 ),
                 **options,
             )
+        elif self.value is torch.from_numpy:
+            if not config.numpy_ndarray_as_tensor:
+                unimplemented(
+                    "torch.from_numpy(). Turn on config.numpy_ndarray_as_tensor to support "
+                    "torch.from_numpy()."
+                )
+            assert len(args) == 1, f"Got arguments {args}"
+            assert not kwargs
+            t = args[0]
+            from .tensor import NumpyTensorVariable
+
+            if isinstance(t, NumpyTensorVariable):
+                # TODO: mark the tensor as non-resizable
+                return wrap_fx_proxy_cls(
+                    target_cls=TensorVariable,
+                    tx=tx,
+                    proxy=tx.output.create_proxy(
+                        "call_function",
+                        torch_np._helpers.ndarrays_to_tensors,
+                        *proxy_args_kwargs(args, {}),
+                    ),
+                    example_value=None,
+                    **options,
+                )
```

Comment:
That's a problem because in Python bytecode, `LOAD_ATTR` needs to load the name from `co_names` in code. Since we don't have `tensor` in `co_names`, `arr.tensor` won't work.